server/service/OperationServerService.js

```diff
@@ -88,27 +89,40 @@ exports.getOperationServerOperationName = function(url) {
  * uuid String 
  * no response value expected for this operation
  **/
-exports.putOperationServerLifeCycleState = function(url, body, uuid) {
+exports.putOperationServerLifeCycleState = function (url, body, uuid) {
   return new Promise(async function (resolve, reject) {
     try {
-      let isUpdated = await fileOperation.writeToDatabaseAsync(url, body, false);
+      let value
+      let oldValue = await OperationServerService.getLifeCycleState(uuid)
+      let remoteServerLifeCycleState = [];
+      let remoteLifeCycleStateEnum = OperationServerService.OperationServerInterfacePac.OperationServerInterfaceConfiguration.lifeCycleStateEnum;
+      for (let remoteLifeCycleStateEnumKey in remoteLifeCycleStateEnum) {
+        if (remoteLifeCycleStateEnum[remoteLifeCycleStateEnumKey] == oldValue || remoteLifeCycleStateEnumKey == oldValue) {
```

Comment:
This should never happen. OpenAPI won't allow any other value than the enum value (never a key).